### PR TITLE
edited build page

### PR DESCRIPTION
**Pull request by GodloveD**

### CODE REVIEW

**Review by jchavezlavalle — COMMENTED** [review]
I think this section needs to be added in the `index.rst` as well, besides that it lgtm @GodloveD

File: build_a_container.rst [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+-------------
+Build options
+-------------
+
+``--builder``
+=============
+
+Singularity 3.0 introduces the option to perform a remote build. The 
+``--builddir`` option allows you to specify a URL to a different build service.
```
Comment (ikaneshiro) [inline build_a_container.rst]: Should this be ``--builder``? Also, does this need to be used in conjunction with `--remote` like `--detached`?

File: build_a_container.rst [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+=============
+
+Build a sandbox (chroot directory) instead of the default SIF format.
+
+``--section``
+=============
+
+Instead of running the entire definition file, only run a specific section or 
+sections.  This option accepts a comma delimited string of definition file
+sections.  Acceptable arguments are: ``setup``, ``post``, ``files``,
```
Comment (ikaneshiro) [inline build_a_container.rst]: Might want to say something along the lines of:
Acceptable arguments include: ``all``, ``none,`` or any combination of valid definition section names. e.g. ``files,environment,post``

File: build_a_container.rst [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+``--update``
+===========
+
+You can build into the same container multiple times (though the
+results may be unpredictable and it is generally better to delete your
+container and start from scratch).
+
+By default if you build using an existing container name container, the 
+``build`` command prompt you to decide whether or not to overwrite the 
+container. Instead of this behavior you can use the ``--update`` option to build
```
Comment (ikaneshiro) [inline build_a_container.rst]: We should mention that the update option is only for sandboxes

**Review by ikaneshiro — CHANGES_REQUESTED** [review]

File: build_a_container.rst (hunk 3 above)
Comment (GodloveD) [reply build_a_container.rst]: Oh I knew we talked about that but didn't realize that was the final decision.  Thank you.

File: build_a_container.rst [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+=============
+
+Build a sandbox (chroot directory) instead of the default SIF format.
+
+``--section``
+=============
+
+Instead of running the entire definition file, only run a specific section or 
+sections.  This option accepts a comma delimited string of definition file
+sections.  Acceptable arguments includ ``all``, ``none`` or any combination of
```
Comment (ikaneshiro) [inline build_a_container.rst:221]: small typo includ->include

**Review by ikaneshiro — APPROVED** [review]

**Review by jchavezlavalle — APPROVED** [review]

File: build_a_container.rst (hunk 4 above)
Comment (jchavezlavalle) [reply build_a_container.rst:221]: I'll fix this in the generation of these docs, thanks @ikaneshiro

**Pull request merged**